### PR TITLE
Adding a new post for Kimai Mobile on IOS

### PR DESCRIPTION
**Pull request by megahed08**
- New post for the Kimai Mobile app IOS release. 
- Image of app screenshots was added and linked in the post.

### CODE REVIEW

File: _posts/2023-08-01-kimai-mobile-ios-release.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: "Kimai Mobile IOS release"
+date: "2023-08-01 20:00:00 +0200"
+author: kevinpapst
```
Comment (kevinpapst) [inline _posts/2023-08-01-kimai-mobile-ios-release.md:4]:
```suggestion
author: cloudrizon
```

**Review by kevinpapst — COMMENTED** [review]
thanks @megahed

**Review by kevinpapst — APPROVED** [review]

**Comment by kevinpapst** [comment]
I expect tests before sending in a PR with such a result
Bildschirmfoto 2023-08-16 um 00 25 08

**Comment by kevinpapst** [comment]
Next time please use https://www.deepl.com/write

Not sure who wrote that, but the excessive use of uppercase words is not common in english

**Comment by megahed08** [comment]
Thanks @kevinpapst for your time and feedback. I will report back to our marketing team to improve our future posts.

**Comment by kevinpapst** [comment]
Thanks 👍

**Pull request merged**